app/jsx/components/TabMetricsComp.jsx

```diff
@@ -3,6 +3,15 @@
 import React from 'react'
 
 class TabMetricsComp extends React.Component {
+  componentWillMount() {
+    if (!(typeof document === "undefined")) {
```

Comment:
@martinhaye  I wasn't sure this was the best way to bring in this JS library. It's working though.